packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -117,15 +145,33 @@ export const PlanDetails = ({
                     id={`plan-price-${interval}`}
                     attrs={{ title: true }}
                     vars={{
-                      amount: getLocalizedCurrency(amount, currency),
+                      amount:
+                        coupon && config.featureFlags.subscriptionCoupons
+                          ? getLocalizedCurrency(
+                              amount ? amount - coupon.amount * 100 : amount,
```

Comment:
The price and the discount are in different units?! 😢

Reply:
yes, although once we hook this up to the back end and have an idea of how we are signaling whether or not the coupon is a fixed amount or percentage, we will need to update this

Reply:
well i should also say that the amount of the plan is in cents essentially so $9.35 is 935, i am not sure how we are going to store the coupon amount. this will get removed once we hook this up to the back end

Reply:
this is just for the dummy data, i'll just remove it and change the coupon to match the units. sorry i wasnt thinking about this too much but this will change anyway when we hook up the back end. we will also have to implement handling fixed amounts vs percentage discounts

Reply:
I believe the backend takes care of that. It looks like regardless of whether the coupon is fixed amount or  percentage, it'll do the calculation for us and return a fixed amount.